src/Analyzer.Utilities/AbstractVersionCheckAnalyzer.cs

```diff
@@ -12,7 +12,7 @@ namespace Analyzer.Utilities
     public abstract class AbstractVersionCheckAnalyzer : DiagnosticAnalyzer
     {
         private const string RuleId = "CA9999";
-        private const string AnalyzerPackageVersion = "2.3.*";
+        private const string AnalyzerPackageVersion = "2.5.*";
```

Comment:
@jasonmalinowski, when is Roslyn planning on moving?

Project-System still won't be able to consume with a hard-check for 2.5 since Roslyn hasn't moved yet.

If this were a min/max check (as AnalysisContextExtensions appears to use), that would work for now.